controllers/events_controller.js

```diff
@@ -124,13 +124,14 @@ module.exports.registerParticipant = async (req, res) => {
 };
 
 module.exports.updateParticipant = async (req, res) => {
-	let { name, email, branch, year, phone } = req.body;
+	let { name, email, branch, year, phone, password } = req.body;
 	let updateObj = {
```

Comment:
because req.files cant be directly inserted in req.body
other ways: use spread for req.body and append image key
Also, creating updateObj reconfirms the fields to required only